app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
```

Comment:
it`s unnecessary to leave pass

Reply:
you can also delete a comment

Reply:
@AnnaBoichenko The same. It is default version. The red area does not exist in my code version.